Content.Client/UserInterface/Systems/Actions/ActionUIController.cs

```diff
@@ -627,22 +627,20 @@ private void OnFilterSelected(ItemPressedEventArgs args)
         QueueWindowUpdate();
     }
 
-    private void OnWindowActionPressed(GUIBoundKeyEventArgs args, ActionButton action)
+    private void OnWindowActionPressed(GUIBoundKeyEventArgs args, ActionButton button)
```

Comment:
```suggestion
    private void OnWindowActionPressed(GUIBoundKeyEventArgs args, ActionButton action)
```
Try to avoid non-functional changes as they cause more merge conflict misery than they're worth.